config/rbac/role.yaml

```diff
@@ -2,8 +2,17 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
+  creationTimestamp: null
```

Comment:
This `creationTimestamp` should not exists, I believe you local `controller-gen` under `/bin` is not up-to-date

Reply:
Damn, I removed this but then rebased and it sneaked in again... Thanks for catching!